app/utils/helpers.ts

```diff
@@ -135,6 +135,9 @@ export function getRoundedTime(value: Moment) {
 
 export function isTablet() {
     const result: SplitViewResult = RNUtils.isRunningInSplitView();
+    if (!result) {
```

Comment:
Can you verify if these errors only occur on Android?

RNUtils is a library we have, and looking at the code, it may return null if there is no context or current activity (that shouldn't happen, but still...). You can see the code here:

`libraries/@mattermost/rnutils/android/src/main/java/com/mattermost/rnutils/helpers/SplitView.kt`

I think that is the right place to solve this, but 0/5.

Reply:
I kind of agree with this, perhaps we should look into the native side and identify why it would be null there.. actually can you share the actual error being reported?

Reply:
This was the error reported

`Cannot read property 'isTablet' of null -> app/utils/helpers.ts in isTablet at line 138:18`